extensions/integration-tests/src/test/notebook.util.ts

```diff
@@ -42,7 +42,8 @@ export const notebookContentForCellLanguageTest: azdata.nb.INotebookContents = {
 	}],
 	metadata: {
 		'kernelspec': {
-			'name': ''
+			'name': '',
```

Comment:
Do these field names need to be quoted here? Same for the other json stuff in this file.

Reply:
Not too sure, I removed the quotes and re-ran the tests and everything seems fine.